l10n_th_vendor_tax_invoice/views/account_view.xml

```diff
@@ -25,4 +25,15 @@
         </field>
     </record>
 
+    <record id="view_tax_form" model="ir.ui.view">
+        <field name="name">account.tax.form.inherit</field>
+        <field name="model">account.tax</field>
+        <field name="inherit_id" ref="account.view_tax_form"/>
+        <field name="arch" type="xml">
+            <field name="cash_basis_base_account_id" position="replace">
```

Comment:
Please try not to use replace first. Or if necessary please set priority.
Please use attributes if possible.
I think you should set it so that,
* Always invisible
* Always not required
The new attrs could be
attrs="{'invisible': [(1, '=', 1)]}"